prometheus-exporters-formula/prometheus-exporters-formula.spec

```diff
@@ -18,13 +18,15 @@
 %define fname prometheus-exporters
 %define fdir  %{_datadir}/susemanager/formulas
 Name:           prometheus-exporters-formula
-Version:        1.0.0
+Version:        1.0.1
 Release:        0
 Summary:        Salt formula for installing and configuring Prometheus exporters
 License:        Apache-2.0
 Group:          System/Packages
 Url:            https://github.com/SUSE/salt-formulas
 Source:         prometheus-exporters-formula-%{version}.tar.gz
+BuildRequires:  python3
```

Comment:
I might not be completely in the picture about our needs at build time here, but shouldn't runtime actually be enough (`Requires`)?

Reply:
At IBS some tests where failing without this. If I recall correctly it was `50-check-installtest`.